swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md

```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
-* The `filter` field - This field will return the data filtered by the ISO Code of a country(`isoCode`).
+* The `filter` field - This field will return the data filtered by the ISO Code of a country(`iso_code`).
 
 ### The `Mutation` type
 The `Mutation` type has a single field.
 * The `add` field - This field will add a given entry to the data source.
 
-## Create the service package
 
-Ballerina uses packages to group code. You need to create a Ballerina package, generate the service code in the package, and write the business logic.
+## Create the service package 
 
->**Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code).
+Ballerina uses packages to group code. Follow the steps below to create a Ballerina package and write the business logic. 
 
-In the terminal, execute the command below to create the Ballerina package for the API implementation.
+> **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
 
-```
-$ bal new covid19 -t service
-```
-
-You view the output below.
-
-```
-Created new package 'covid19' at covid19.
-```
+1. In the Terminal, execute the command below to create the Ballerina package for the API implementation.
 
-This creates a directory named `covid19` with the default module along with a sample code for the service as shown below.
-
-```
-.
-├── covid19
-│   ├── tests
-│   │   └── service_test.bal
-│   ├── Ballerina.toml
-│   └── service.bal
-```
-
-## Create the data source 
+    ```
+    $ bal new write-a-graphql-api
+    ```
 
-Before writing the GraphQL service, let's create a data source for the project. This will mimic a database that stores the data for the service using an in-memory table in Ballerina as the data source.
+    You should see the output similar to the following.
 
-### Define the types for the data source
+    ```
+    package name is derived as 'write_a_graphql_api'. Edit the Ballerina.toml to change it.
 
-To define the data type to use in the table, replace the `service.bal` file with the code below. 
+    Created new package 'write_a_graphql_api' at /Users/write-a-graphql-api.
+    ```
 
+    This creates a directory named `write-a-graphql-api` with the default module and a sample code for the service, as shown below. 
 
-```ballerina
-public type CovidEntry record {|
-    readonly string isoCode;
-    string country;
-    decimal cases?;
-    decimal deaths?;
-    decimal recovered?;
-    decimal active?;
-|};
-```
-
-In this code:
-- The record type `CovidEntry` is defined.
-- A single entry will include the ISO code for the location (`isoCode`), name of the country (`country`), number of cases (`cases`), number of deaths (`deaths`), number of recovered cases (`recovered`), and the number of active cases (`active`).
+    ```
+    .
+    │   ├── Ballerina.toml
+    │   └── main.bal
+    ```
```

Comment:
```suggestion
    ```
    .
    │   ├── Ballerina.toml
    │   └── main.bal
    ```
```
Shall we add the package name as well.

Reply:
Will update after changing the package names as per https://github.com/ballerina-platform/ballerina-dev-website/pull/8664#discussion_r1496724115.